pantheon-module-report

```diff
@@ -26,7 +29,27 @@ if [ "$VERBOSITY" == "" ]; then
   esac
 fi
 
-sites=$(terminus site:list --format=list --field=name --filter="frozen!=1" | sort)
+ORG=$3
+if [ "$ORG" == "" ]; then
```

Comment:
Additionally `==` only works in bash. If you have to drop down to just posix shell `=` is required as `==` will throw an error. It's typically best to use double bracket `[[ ... ]]` when doing tests in bash as well.

Obviously since `#!` points at bash, we should be fine. Just some gotchas on environments where sh is the only thing available or bash is an alias for sh.

Reply:
thanks, I don't really know much about bash scripting so I made some of these revisions but didn't want to go too crazy updating everything.